clcache.py

```diff
@@ -1528,6 +1579,8 @@ def processCompileRequest(cache, compiler, args):
     except CalledForPreprocessingError:
         printTraceStatement("Cannot cache invocation as {}: called for preprocessing".format(cmdLine))
         updateCacheStatistics(cache, Statistics.registerCallForPreprocessing)
+    except IncludeNotFoundException:
+        pass
```

Comment:
Could you add a simple tracing message here? I just run into this and did not have a clue, why there are no hits.